styles/styles.css

```diff
@@ -427,6 +428,13 @@ main .section {
   margin: 2rem 0;
 }
 
+main .section .section-container {
+  max-width: 81.25rem;
+  margin-inline: auto;
+  padding-left: 16px;
+  padding-right: 16px;
+}
+
 main .section.padded > .section-container {
```

Comment:
The padded selector needs to be restructured. I'll take out the padding selector.